client/components/domains/register-domain-step/analytics.js

```diff
@@ -7,24 +7,32 @@ import {
 
 const noop = () => {};
 
-export const recordMapDomainButtonClick = ( section ) =>
+export const recordMapDomainButtonClick = ( section, flowName ) =>
 	composeAnalytics(
 		recordGoogleEvent( 'Domain Search', 'Clicked "Map it" Button' ),
-		recordTracksEvent( 'calypso_domain_search_results_mapping_button_click', { section } )
+		recordTracksEvent( 'calypso_domain_search_results_mapping_button_click', {
+			section,
+			flow_name: flowName,
+		} )
 	);
 
-export const recordTransferDomainButtonClick = ( section, source ) =>
+export const recordTransferDomainButtonClick = ( section, source, flowName ) =>
 	composeAnalytics(
 		recordGoogleEvent( 'Domain Search', 'Clicked "Use a Domain I own" Button' ),
-		recordTracksEvent( 'calypso_domain_search_results_transfer_button_click', { section, source } )
+		recordTracksEvent( 'calypso_domain_search_results_transfer_button_click', {
+			section,
+			source,
+			flow_name: flowName,
+		} )
 	);
 
-export const recordUseYourDomainButtonClick = ( section, source ) =>
+export const recordUseYourDomainButtonClick = ( section, source, flowName ) =>
```

Comment:
There are some instances of this function that are being called with only one parameter, we should eventually fix those in a future PR.